docs/CONTRIBUTING.md

```diff
@@ -16,30 +16,58 @@ please [see here](http://bit.ly/argo-wf-cmty-mtng).
 
 ## Contributor Meetings (twice monthly)
 
-A weekly opportunity for committers and maintainers of Workflows and Events to discuss their current work and
-talk about what’s next. Feel free to join us! For Contributor Meeting information, minutes and recordings
-please [see here](https://bit.ly/argo-data-weekly).
+An opportunity for contributors and maintainers of Workflows and Events to discuss their current work and talk about what’s next. Feel free to join us!
+See the [Contributor Meeting doc](https://bit.ly/argo-data-weekly) for minutes, recordings, and more information.
+
+## Slack
+
+You can join the following channels on [CNCF Slack](https://argoproj.github.io/community/join-slack):
+
+* `#argo-workflows`: discussions focused mainly on use of Argo Workflows
+* `#argo-wf-contributors`: discussions focused mainly on development of Argo Workflows
+
+## Roles
+
+The Argo project currently has 4 designated [roles](https://github.com/argoproj/argoproj/blob/main/community/membership.md):
+
+* Member
+* Reviewer
+* Approver
+* Lead
+
+The Reviewer and Approver roles can optionally be scoped to an area of the code base (for example, UI or docs).
+
+Current roles for Reviewers and above can be found in [OWNERS](../OWNERS).
+
+If you are interested in formally joining the Argo project, [create a Membership request](https://github.com/argoproj/argoproj/issues/new?template=membership.md&title=REQUEST%3A%20New%20membership%20for%20%3Cyour-GH-handle%3E) in the [argoproj](https://github.com/argoproj/argoproj) repository as described in the [Membership](https://github.com/argoproj/argoproj/blob/main/community/membership.md) guide.
```

Comment:
```suggestion
If you are interested in formally joining the Argo project, [create a Membership request](https://github.com/argoproj/argoproj/issues/new?template=membership.md&title=REQUEST%3A%20New%20membership%20for%20%3Cyour-GH-handle%3E) in the [`argoproj` repository](https://github.com/argoproj/argoproj) as described in the [Membership](https://github.com/argoproj/argoproj/blob/main/community/membership.md) guide.
```

we could put `argoproj` in backticks, and then we could remove it from the `.spelling`. nbd either way though, up to you

Reply:
I'll just leave as is